src/components/Exchange/PositionEditor.js

```diff
@@ -465,6 +502,33 @@ export default function PositionEditor(props) {
   };
   const ERROR_TOOLTIP_MSG = {
     [ErrorCode.InvalidLiqPrice]: t`Liquidation price would cross mark price.`,
+    [ErrorCode.InsufficientCollateral]: (
+      <>
+        <p>
+          <Trans>The Deposit amount is insufficient to bring Net Value below max allowed leverage of 100x.</Trans>
+        </p>
+        <br />
+        <StatsTooltipRow
+          label={t`Current Net Value`}
+          value={netValueAfterClosingFee && formatAmountWithDollarSign(netValueAfterClosingFee, USD_DECIMALS, 2, true)}
+          showDollar={false}
+        />
+        <StatsTooltipRow
+          label={t`Min. Net Value needed is`}
+          value={minimumCollateralRequired && formatAmount(minimumCollateralRequired, USD_DECIMALS, 2, true)}
+          showDollar={true}
+        />
+        {minimumCollateralToDeposit && minimumCollateralToDepositStr && (
+          <StatsTooltipRow
+            label={t`Deposit at least`}
+            value={[
+              formatAmountWithDollarSign(minimumCollateralToDeposit, USD_DECIMALS, 2, true),
+              minimumCollateralToDepositStr,
```

Comment:
not sure, but probably would be better to keep string formatting in the jsx here instead of str. 209, just to keep business logic and layout separately